src/portal/src/i18n/lang/fr-fr-lang.json

```
@@ -80,26 +80,26 @@
         "PASSWORD": "Le mot de passe doit être composé de 8 à 128 caractères avec au moins 1 majuscule, 1 minuscule et 1 chiffre.",
         "CONFIRM_PWD": "Les mots de passe ne correspondent pas.",
         "SIGN_IN_USERNAME": "Le nom d'utilisateur est obligatoire.",
-        "SIGN_IN_PWD": "Le mot de passe obligatoire.",
+        "SIGN_IN_PWD": "Le mot de passe est obligatoire.",
         "SIGN_UP_MAIL": "L'email n'est utilisé que pour réinitialiser votre mot de passe.",
         "SIGN_UP_REAL_NAME": "Prénom et Nom",
         "ITEM_REQUIRED": "Le champ est obligatoire.",
-        "SCOPE_REQUIRED": "Le champ est obligatoire et doit  être au format scope.",
+        "SCOPE_REQUIRED": "Le champ est obligatoire et doit être au format scope.",
         "NUMBER_REQUIRED": "Le champ est obligatoire et doit être numérique.",
         "PORT_REQUIRED": "Le champ est obligatoire et doit être un numéro de port valide.",
         "CRON_REQUIRED": "Le champ est obligatoire et doit être au format cron.",
         "EMAIL_EXISTING": "L'adresse e-mail existe déjà.",
```

You might use the same word email or e-mail along side of traduction
```suggestion
        "EMAIL_EXISTING": "L'adresse email existe déjà.",
```